docs/videos/EventBusVideo/eventBus.vtt

```diff
@@ -0,0 +1,110 @@
+Any data that would be useful to share between systems as soon it's available could be packaged into an event.
+
+7
+00:00:40.000 --> 00:00:42.600
+Examples of things that could be events include:
+
+8
+00:00:42.700 --> 00:00:48.700
+medical claim processing information, contact center ticket information, or Veteran personal data updates. 
+
+9
+00:00:51.000 --> 00:00:55.900
+Currently, many VA systems poll for data infrequently, and between many layers of systems. 
+
+10
+00:00:56.000 --> 00:01:05.200
+This means an update to one system may take days or months to be updated in all locations, as the data is slowly requested by each system from another on their own schedules. 
+
+11
+00:01:05.700 --> 00:01:14.100
+Not only that, but not all systems are connected or get data from the same source; something that is updated in one place may not ever reach other systems.
+
+12
+00:01:17.500 --> 00:01:20.000
+Event Bus offers a solution to these problems. 
+
+13
+00:01:20.200 --> 00:01:29.000
+By providing a system that reads from sources and offers the data in a specialized output stream in real-time, client systems can stay connected and receive data as it is available.
+
+14
+00:01:34.000 --> 00:01:39.400
+This reduces the need for slow, inter-system polling and one-to-one connections between many dependent systems. 
+
+15
+00:01:43.200 --> 00:01:49.600
+If a Veteran updates their address in one source, it can be fed into Event Bus and all systems can be notified at once, as it happens. 
+
+16
+00:01:50.000 --> 00:01:54.200
+No more slow propagation and compiling data from many other systems!
+
+17
+00:01:56.200 --> 00:01:59.100
+Event Bus also offers further advantages.
+
+18
+00:01:59.500 --> 00:02:08.500
+By using an event streaming platform called Kafka and providing a high-uptime single source of data, Event Bus sidesteps some of the reliability issues other systems have. 
+
+19
+00:02:08.700 --> 00:02:15.700
+Kafka temporarily keeps all events in its output stream and also tracks which event each client last consumed.
+
+20
+00:02:15.900 --> 00:02:22.500
+If there is any interruption in the connection, the client can resume reading from the data stream as soon as they reconnect, right where they left off.
+
+21
+00:02:23.100 --> 00:02:30.200
+In addition, Event Bus is a centralized hub for events, reducing intersystem dependencies and the number of intersystem connections needed.
+
+22
+00:02:31.100 --> 00:02:35.200
+With Event Bus acting as the Data Hub, systems don't need to be so tightly coupled!
+
+23
+00:02:38.300 --> 00:02:47.000
+As of August 2024, Event Bus is fully deployed as a production service on the Lighthouse Delivery Infrastructure (LHDI) platform. 
+
+24
+00:02:47.500 --> 00:02:57.300
+We currently have a Decision Letter Availability event available, and the Benefits Management Tools team is working on an app to consume that event to trigger an email notification to Veterans. 
+
+25
+00:02:57.700 --> 00:03:05.900

```

Comment:
I'm a little wary of showing MPI, Burn Pit, and Enrollment System on this slide, given that we haven't talked to these teams and that those consumers are speculative.

Suggestions:
* Update the script to say something about connections shown with solid lines are in place or on the way, while dotted lines are potential future clients. 
* Also put some text on the slide itself that says the same thing about solid lines and dotted lines.

Reply:
Seems Anthony just wants to nix the whole section, so I think I'll just crop out that part, if I can.

Reply:
Yeah! That makes it easy.

Reply:
Removed that section from the video and the transcript.